perceval/backends/core/github.py

```diff
@@ -85,6 +86,31 @@ def __init__(self, owner=None, repository=None,
                                    sleep_for_rate, min_rate_to_sleep)
         self._users = {}  # internal users cache
 
+    def __get_issue_comments(self, issue_number):
+        """ Get issue comments """
+        comments = []
+        group_comments = self.client.get_paginated_items("/issues/" + str(issue_number) + "/comments", "comment")
+        for raw_comments in group_comments:
+            self._push_cache_queue('{COMMENTS}')
+            self._push_cache_queue(raw_comments)
+            self._flush_cache_queue()
+            for comment in json.loads(raw_comments):
+                comment["user_data"] = self.__get_user(comment['user']['login'])
+                comments.append(comment)
+
+        return comments
+
+    def __get_issue_assignees(self, raw_assignees):
```

Comment:
This should be removed and included in another PR. It's a new feature not related with this PR.